frontend/src/static/js/v_summary.js

```diff
@@ -212,7 +213,8 @@ window.vSummary = {
 
       if (hash.timeframe) { this.filterTimeFrame = hash.timeframe; }
       if (hash.mergegroup) {
-        this.isMergeGroup = convertBool(hash.mergegroup);
+        // make a copy to prevent custom merged groups from overwritten
+        this.customMergedGroups = hash.mergegroup;
```

Comment:
the mapping of hash values to the `group name -> isMerge` map can be done here

Reply:
it doesn't matter whether `this.filtered` is initialised or not right? Here is just to populate the data from `hash.mergegroup` to `mergedGroupsMap`?

Reply:
But the keys of `mergedGroupsMap` need to be the group names, which comes from `this.filtered`. `hash.mergegroup` only has the merged groups' names, not the unmerged ones.

Reply:
Ahh i see.